docs/how-to/namespace-transition.md

```diff
@@ -174,24 +410,18 @@ release from the `knative-gcp` repository. We will update the page with specific
      ...
    ```
 
-1. Delete the old BrokerCell since it is no longer useful.
+1. Delete the old BrokerCell since it is no longer used. This will not cause any
+   service disruptions since the previous step already verified that the new
+   BrokerCell is ready and active.
 
    ```shell script
-   kubectl delete deployment default-brokercell-fanout -n cloud-run-events
-   kubectl delete deployment default-brokercell-ingress -n cloud-run-events
-   kubectl delete deployment default-brokercell-retry -n cloud-run-events
-   kubectl delete service default-brokercell-ingress -n cloud-run-events
-   kubectl delete horizontalpodautoscaler.autoscaling/default-brokercell-fanout-hpa -n cloud-run-events
-   kubectl delete horizontalpodautoscaler.autoscaling/default-brokercell-ingress-hpa -n cloud-run-events
-   kubectl delete horizontalpodautoscaler.autoscaling/default-brokercell-retry-hpa -n cloud-run-events
-   # optional: remove post-install for v0.17.6
-   kubectl delete job.batch/storage-version-migration-knative-gcp -n cloud-run-events
+   kubectl delete brokercell default -n cloud-run-events
    ```
 
-1. For each existing source delete the old source deployment. A new source
-   deployment will be created automatically. The example below illustrates the
-   process for a Cloud PubSub source `cloudpubsubsource-test` in the `example`
-   namespace.
+1. **If you have sources deployed.** For each existing source delete the old
```

Comment:
Can we provide a command to list all sources in the cluster? Probably `kubectl get sources --all-namespaces` should work. We should also explain how to identify the ones that are GCP.

Reply:
Updated.